company-profiles/onna.md

```diff
@@ -0,0 +1,37 @@
+# Onna
+
+## Company blurb
+
+We launched Onna to solve one of the monumental data challenges of our time: how to make everybody's proprietary 
+information accessible, useful, and private. Large organizations use on average 175 apps for daily work tasks. But as 
+these apps proliferate, they easily become separate, siloed stores of data. Our biggest goal is to help business and IT 
+leaders make sense of the vast amounts of information in their organizations and turn that information into knowledge.
+
+We believe that software should allow us to be productive and knowledgeable with significant competitive advantage to be 
+gained for those that can achieve both. We also believe that the ability to secure your own data and keep it private, 
+should be guaranteed at all times.
```

Comment:
yeah I always cut my lines in a `.md` file at around 120 characters so that its readable from without having to scroll, or otherwise having the editor artificially introduce linebreaks. This still renders nicely on a webpage. I guess I can make it all two really long lines of text.

Reply:
Thanks @bkimbrough88, good to know. I didn't preview but I'll know in future this is possible.